app/src/components/button/Button.tsx

```diff
@@ -5,23 +5,20 @@ import {
 } from "react-aria-components";
 import { css } from "@emotion/react";
 
-import {
-  SizingProps,
-  StylableProps,
-  VarianceProps,
-} from "@phoenix/components/types";
+import { SizingProps, StylableProps, Variant } from "@phoenix/components/types";
 
 import { buttonCSS } from "./styles";
 
-interface ButtonProps
-  extends AriaButtonProps,
-    SizingProps,
-    VarianceProps,
-    StylableProps {
+interface ButtonProps extends AriaButtonProps, SizingProps, StylableProps {
   /**
    * An optional prefixed icon for the button
    */
   icon?: ReactNode;
+  /**
+   * The variant of the button
+   * @default: 'default'
+   */
+  variant?: Exclude<Variant, "info">;
```

Comment:
Generally agree this interface can be removed and is not helpful (e.g. a bad abstraction). We can move it into the button itself as a prop.